Automattic/wp-calypso
# Layout: Replace `LayoutLoggedOut` and `LayoutLoggedOutOAuth` with `Layout`

Fixes #2960. The layout changes based on the current section and presence of a `user` prop.

`LayoutLoggedOutDesign` hasn't been removed yet because it is also rendered on the server and has dependencies (like `abtest`) that assume a browser environment. I started working on this, but updating `Layout` to run on the server is a rather large task, and will be handled separately - I'll open an issue for this once this is merged.

**Testing**
There are no visual changes. We need to assert that the layout doesn't break when the user is logged out, logged in, using the desktop app, and on `/design`.

*Logged out*
- Visit `/start` while logged out, and assert that the page isn't broken.
- Go through the signup flow an ensure the email verification notice shows.

*Logged in*
- Visit any page while logged in, and assert that the page isn't broken.

*Using the desktop app*
- Check out this branch inside the `calypso/` directory in the desktop app (`cd calypso && git checkout update/one-layout`).
- `make run` from `wp-desktop/` and assert that the login page isn't broken.
- Log in and assert that the Reader (or any other page, once logged in) isn't broken.

*`/design`*
- Visit `/design` while logged out and assert that the page isn't broken.
